misc/experimental/rust/celix_bindings/build.rs

```diff
@@ -44,19 +58,15 @@ fn main() {
     println!("cargo:info=Start build.rs for celix_bindings");
     let include_paths = print_include_paths().unwrap();
 
```

Comment:
Misisng `cargo:rustc-link-lib=` leads to no DT_NEEDED for `framework` and `utils` shared objects.